src/applications/vaos/new-appointment/redux/selectors.js

```diff
@@ -342,7 +340,7 @@ export function getFacilityPageInfo(state) {
     parentOfChosenFacility: getParentOfChosenFacility(state),
     cernerOrgIds: selectCernerOrgIds(state),
     isCernerOnly: selectIsCernerOnlyPatient(state),
-    siteId: getSiteIdFromOrganization(getChosenParentInfo(state)),
+    siteId: getChosenParentInfo(state)?.id,
```

Comment:
Also sort of misnamed, but seems to only be used for an error message fallback in the old facilities page